src/compute-client/src/controller/instance.rs

```diff
@@ -1536,6 +1557,20 @@ where
         ))
     }
 
+    fn handle_copy_to_response(
+        &mut self,
+        sink_id: GlobalId,
+        response: CopyToResponse,
+    ) -> Option<ComputeControllerResponse<T>> {
+        // We might not be tracking this COPY TO because we have already returned a response.
+        // In that case, we ignore the response.
```

Comment:
This is in the case of another replica having already completed the copy?

Reply:
Yes! Updating the comment to clarify